backend/config/sync/user-role.admin.json

```diff
@@ -36,6 +48,18 @@
     {
       "action": "api::event.event.update"
     },
+    {
```

Comment:
I wonder why there are two of the same announcement apis could you delete one?

Reply:
I removed one of them